src/components/Review/ReviewSectionRowLastActionDate.tsx

```diff
@@ -16,15 +16,15 @@ const ReviewSectionRowLastActionDate: React.FC<ReviewSectionComponentProps> = ({
         return (
           <LastDate
             title={strings.ACTION_DATE_REVIEW_STARTED}
-            indicator={getSimplifiedTimeDifference(thisReview?.timeStatusCreated)}
+            indicator={getSimplifiedTimeDifference(thisReview?.stage.statusCreatedDate)}
```

Comment:
i think this is different now, as previously it was date review was started, and I think stage.statusCreatedDate is when application jumped to new stage

Reply:
Ahh i see you've changed to putting statusCreated together with stage, which is a bit confusing for me, I think the stage no ReviewDetails should be explicitly 'applicationStage' -> to avoid the confusion, and maybe keep review status related field within the review, rather then stage ?

Reply:
Yes... I can see it might be a bit confusing indeed. Before we had a composed property:
```

Reply:
Will be changing this to the above suggestion, using fields for times more aligned with how is stored in `review` table:
interface StageAndStatus {
  stage: StageDetails
  status: ApplicationStatus
  timeStageCreated: Date
  timeStatusCreated: Date 
}
```